CHANGELOG.md

```diff
@@ -1,5 +1,91 @@
 # Changelog
 
+## v1.8.0-rc4
+
+Summary of Changes
+------------------
+
+**Minor Changes:**
+* api/v1: Add observation proto enum (Backport PR #12173, Upstream PR #12085, @glibsm)
+* docs: Extend BPF-based masquerading section (Backport PR #12203, Upstream PR #12145, @brb)
+* helm: Remove affinity from cilium-etcd-operator (Backport PR #12173, Upstream PR #12139, @brb)
+* Update the Cilium eBPF library to the latest version. (Backport PR #12103, Upstream PR #12068, @tklauser)
+
+**Bugfixes:**
+* bpf: Use `nproc --all` for __NR_CPUS__ (Backport PR #12173, Upstream PR #12121, @gandro)
```

Comment:
Minor: I think we may want to escape the markdown formatting or wrap it in backticks.

Reply:
Good spotting.